test/unit/modules/transactions.js

```diff
@@ -417,9 +610,7 @@ describe('transactions', function () {
 				});
 			});
 
-			it.skip('should get transaction with outtransfer asset for transaction id', function (done) {
```

Comment:
Test from line 595 is skipped. Why?

Reply:
These tests were written when lisk-js didn't correctly create the inTransfer and outTransfer transactions. But now it does, so we should add these tests. That being said, I think there should be a new PR for those tests and not this one.

Reply:
there are not existing examples of transaction types 6 and 7 in prepared mocks